ansible_playbook/vars/biothings_studio.yml

```diff
@@ -17,7 +17,7 @@ software:
       elasticsearch_yml:
         - "node.data: true"
         - "node.master: true"
-    studio_repository: "https://github.com/biothings/{{ app_name }}.git" 
+    biothings_repository: "{{ biothings_repository | default('https://github.com/biothings/biothings.api.git') }}"
```

Comment:
I think the biothings_repository should be the repo of Biothings Studio, shouldn't it ?

Reply:
Actually, `studio_repository` is unused variable on this Ansible setup.
Instead, Ansible clones the biothings.api repo to override the requirement_web.txt version, so I use `biothings_repository ` for customizing repo url https://github.com/newgene/biothings_docker/pull/2/files#diff-cbe1e6b10e994fa5eff7d0a84cde5413022c754c6b61a7b8293ebbf0c117c5e7L103